src/background.js

```diff
@@ -49,20 +51,24 @@ if ('commands' in chrome) {
 
 // *************************************************************************
 {
-  const onInstall = ({reason}) => {
+  const onInstall = ({
+    reason
+  }) => {
     chrome.runtime.onInstalled.removeListener(onInstall);
     const manifest = chrome.runtime.getManifest();
     // Open FAQs page once after installation to guide new users.
     // Do not display it in development mode.
-    if (reason == 'install' && manifest.update_url) {
+    if (reason === 'install' && manifest.update_url) {
       setTimeout(openURL, 100, {
         url: `http://add0n.com/stylus.html?version=${manifest.version}&type=install`
       });
     }
     // reset L10N cache on UI language change or update
-    const {browserUIlanguage} = tryJSONparse(localStorage.L10N) || {};
+    const {
+      browserUIlanguage
```

Comment:
This robotic expansion of literals doesn't make the code more readable. On the contrary, it's less readable in such cases as this one where only one parameter is used.